README.md

```diff
@@ -23,6 +23,23 @@ conda install -c conda-forge sleepecg
 The [contributing guide](https://github.com/cbrnr/sleepecg/blob/main/CONTRIBUTING.md) contains detailed instructions on how to contribute to SleepECG.
 
 
+## Dataset reading
```

Comment:
Could we also briefly list the available readers for the various ECG databases here?